packages/teams-js/src/private/store.ts

```diff
@@ -39,14 +40,25 @@ export enum StoreDialogType {
   AppDetail = 'appdetail',
 }
 
+/**
+ * @beta
+ * @hidden
+ * Interface of store dialog size
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export interface StoreSizeInfo {
+  size?: DialogSize;
+}
+
 /**
  * @beta
  * @hidden
  * Interface of open full store, copilot store and in-context-store function parameter
  * @internal
  * Limited to Microsoft-internal use
  */
-export interface OpenFullStoreAndICSParams {
+export interface OpenFullStoreAndICSParams extends StoreSizeInfo {
```

Comment:
If I remember correctly, this interface can be used to either open the full store (a full screen experience) or the ICS store (a sort of dialog).

I don't think developers should be allowed to set a dialog size if they are opening a full store, that value wouldn't have any meaning. Do you agree?

Reply:
Actually a 'full store' is also a dialog, which means opening a store without navigation to a specific collection in a dialog